src/message.rs

```diff
@@ -147,13 +147,18 @@ impl MsgId {
 
     /// Put message into trash chat and delete message text.
     ///
-    /// It means the message is deleted locally, but not on the server
-    /// yet.
+    /// It means the message is deleted locally, but not on the server.
+    /// We keep some infos to
+    /// 1. not download the same message again
+    /// 2. be able to delete the message on the server if we want to
     pub async fn trash(self, context: &Context) -> crate::sql::Result<()> {
         let chat_id = ChatId::new(DC_CHAT_ID_TRASH);
         context
             .sql
             .execute(
+                // If you change which information is removed here, also change delete_expired_messages() and
+                // which information dc_receive_imf::add_parts() still adds to the db if the chat_id is TRASH
+                // (to find the exact location, you can grep for `MsgId::trash()`)
```

Comment:
Do you propose to grep for `MsgId::trash` in a comment inside `MsgId::trash`?

Reply:
Well, yes :)
- I propose the same at other places, where I want to hint to this part.
- And if you grep for `MsgId:trash` now, you will find all three places that have to be changed.

If you think it's more confusing than it helps, I'll remove it :)

Reply:
maybe it is clearer to remove the 3rd line here and at the other two places. `MsgId::trash` might also be used in comments elsewhere in the future, thinks may be even unclearer then.  
the first two lines are already very precise (thanks for that!) and point to the correct functions.